addons/hr_attendance/static/src/components/hr_presence_status/hr_attendance_presence_status.js

```diff
@@ -0,0 +1,26 @@
+import { HrPresenceStatus } from "@hr/components/hr_presence_status/hr_presence_status";
+import { patch } from "@web/core/utils/patch";
+import { _t } from "@web/core/l10n/translation";
+import { hrPresenceStatus } from "../../../../../hr/static/src/components/hr_presence_status/hr_presence_status";
+
+patch(HrPresenceStatus.prototype, {
+    get label() {
+        if (this.value === 'presence_present') {
+            const hasUser = this.props.record.data.user_id;
+            if (!hasUser) {
+                return _t("Present (untracked)");
+            }
+        } else if (this.value === 'presence_out_of_working_hour') {
+            return _t("Off-Hours");
+        }
+        
+        return super.label;
+    }
+});
+
+Object.assign(HrPresenceStatus, {
```

Comment:
```suggestion
Object.assign(hrPresenceStatus, {
```
hrPresenceStatus is an object; HrPresenceStatus is a class :)